src/CashuWallet.ts

```diff
@@ -72,29 +87,48 @@ class CashuWallet {
 		return fee;
 	}
 
-	static getDecodedLnInvoice(invoice: string) {
+	static getDecodedLnInvoice(invoice: string): {
```

Comment:
lol. i guess we don't really need this function. should we remove it and call `decode` directly?

Reply:
should we export it in index.ts ?
```typescript
import { CashuMint } from './CashuMint.js';
import { CashuWallet } from './CashuWallet.js';
import { Proof } from './model/Proof.js';
import { getEncodedToken, getDecodedToken, deriveKeysetId } from './utils.js';
import { decode as getDecodedLnInvoice } from '@gandlaf21/bolt11-decode';
export {
	CashuMint,
	CashuWallet,
	Proof,
	getDecodedToken,
	getEncodedToken,
	deriveKeysetId,
	getDecodedLnInvoice
};
```